### DISCUSSION TITLE
Countdown Timer With Restart Timer Button

### DISCUSSION THREAD

**Discussion by richie19rich77** [Windos/BurntToast#235] [Q&A]
Hi All,

I have created a script to prompt the user for their login credentials which then maps a drive.  The drive will stay connected for 20 minutes with a notification being sent at 5 minute intervals.

That works fine, but to reset the timer using the button I am having to execute another powershell script when the button is pressed, passing the PID of the original powershell process, killing the original powershell and starting a new countdown and new 20 minute notification.

Is there an easier way to run a powershell command linked to a button, rather than "powershell://run" command?

Thanks

**Comment by richie19rich77** [comment]
Sorry ignore post.  Just read https://github.com/Windos/BurntToast/discussions/140 and have implemented PowerShell 7 to get this working.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
